docs/dev-setup.md

```diff
@@ -50,3 +50,24 @@ install the operator-sdk tools.
     ```
     oc apply -f deploy/crds/metalkube_v1alpha1_baremetalhost_cr.yaml
     ```
+
+## Using libvirt VMs with Ironic
+
+In order to use VMs as hosts, they need to be connected to vmbc and
```

Comment:
nit: s/vmbc/vbmc/ (also maybe link to virtualbmc docs?)

Reply:
Done